src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -89,6 +89,37 @@ public class Ado : NotificationsBase, IAdo {
         return errorCodes.Any(errorStr.Contains);
     }
 
+    private static async Async.Task<OneFuzzResultVoid> ValidatePath(string project, string path, TreeStructureGroup structureGroup, WorkItemTrackingHttpClient client) {
+        var pathParts = path.Split('\\');
+        if (pathParts[0] != project) {
+            return OneFuzzResultVoid.Error(ErrorCode.ADO_VALIDATION_INVALID_PATH, new string[] {
+                $"Path \"{path}\" is invalid. It must start with the project name, \"{project}\".",
+                $"Example: \"{project}\\{path}\".",
+            });
+        }
+
+        var current = await client.GetClassificationNodeAsync(project, structureGroup, depth: pathParts.Length - 1);
```

Comment:
Just to check my understanding: for depth, we're doing parts.Length - 1 because we're considering the last part in the path as a leaf and we don't care about its children (if it has any) right?

Reply:
`parts.Length - 1` because the project is the root node, so we don't need to count it for the child-depth.